Added WindCORES to the list

DonHege opened the pull request:
I added WindCORES to the list, from my point of view a quite interesting project. They place small datacenters inside the tower of wind turbines, bringing the consumers of electricity closer to the producers while making the entire IT infrastructure more decentralized at the same time.

Commit:
Added WindCORES to the list

nglgzz commented on README.md:
```diff
@@ -163,6 +163,9 @@ Short description.
 
+- [WindCORES](https://www.windcores.de/en/homepage/) | Germany | 2018
+
+  A subsidiary of Westfalenwind, a German wind energy developer. They host servers inside wind turbines, bringing the consumer of electricity closer to the producer.
```
It took me a bit to realize what they actually do until I came across the term data center provider, so I think it would be a good idea to add it to the description.

```suggestion
  A subsidiary of Westfalenwind, a German wind energy developer. They are a data center provider that hosts servers inside wind turbines, bringing the consumer of electricity closer to the producer.
```

DonHege committed:
Added clarification for WindCORES

Integrated the clarification for WindCores (that they are a datacenter provider).

Co-Authored-By: Angelo Gazzola <angelo@nglgzz.com>

DonHege committed:
Merge branch 'master' into master

DonHege commented:
Updated branch to make it mergable into master.

nglgzz commented:
Thabo you for your contribution @DonHege :sparkles:
